deployment/migration-assistant-solution/lib/eks-infra.ts

```diff
@@ -179,7 +179,15 @@ export class EKSInfra extends Construct {
                     's3:GetObject',
                     's3:PutObject',
                     's3:ListBucket',
+                    's3:ListAllMyBuckets',
                     's3:DeleteObject',
+                    "s3:DeleteObjectVersion",
+                    "s3:ListBucketVersions",
+                    "s3:ListBucketMultipartUploads",
+                    "s3:AbortMultipartUpload",
```

Comment:
lol - what if you didn't have permissions to do this?  Would you have to finish your upload?